packages/js/components/src/split-dropdown/split-dropdown.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { cloneElement, createElement } from '@wordpress/element';
+import { Icon, chevronDown, chevronUp } from '@wordpress/icons';
+import { Button, ButtonGroup, Dropdown } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+export type SplitDropdownProps = {
+	className?: string;
+	disabled?: boolean;
+	variant?: string;
+	children: JSX.Element[];
+};
+
+export const SplitDropdown: React.FC< SplitDropdownProps > = ( {
+	className = '',
+	children,
+	disabled,
+	variant = 'primary',
+}: SplitDropdownProps ) => {
+	const groupActionProps = Object.assign(
+		{ variant },
+		disabled ? { disabled } : {}
+	);
+	const mainActionProps = {
+		...groupActionProps,
+		className: `woocommerce-split-dropdown__main-button ${ className }`,
+	};
+	const [ mainAction, ...menuActions ] = children;
+	const menuIcon = chevronDown;
```

Comment:
Do we need extra `const`s for this?  Could we directly use `chevronUp` and `chevronDown` below?

Reply:
We don't; it was just the shortest distance from having them as props to having them hardcoded (and incidentally the shortest distance back if we ever needed to make them configurable later), but since I'm going to make some changes anyway I will go ahead and remove the extraneous declarations.